packages/rlc-common/src/metadata/buildPackageFile.ts

```diff
@@ -94,23 +94,30 @@ function restLevelPackage(model: RLCModel) {
     bugs: {
       url: "https://github.com/Azure/azure-sdk-for-js/issues"
     },
-    files: [
-      "dist/",
-      generateTest ? "dist-esm/src/" : "dist-esm/",
-      `types/${packageDetails.nameWithoutScope ?? packageDetails.name}.d.ts`,
-      "README.md",
-      "LICENSE",
-      "review/*"
-    ],
+    files:
+      model.options?.moduleKind === "esm"
```

Comment:
Same, there should be no reason we would ever need to use CJS or ESM determinations if we just use tshy to generate all of the builds for us.

Reply:
Unless this is for legacy code and we're not switching over yet

Reply:
Yeah, I don't want to change the way autorest generates RLC, at least for now. There may also be external customers interested in this, for example, someone can generate an RLC without metadata inside their project instead of making it an external package. In this scenario, if they are not using ESM they'd be forced to migrate their project which may not be feasible.